source/content/solr-drupal-9.md

```diff
@@ -0,0 +1,147 @@
+
+- Install [Composer](https://getcomposer.org/)
+
+- Create a Drupal 9 with Integrated Composer managed site on Pantheon following the [Drupal 9](/drupal-9) guide, or migrate to Drupal 9 with Integrated Composer using the (Drupal 9 Migration Guide)[/guides/drupal-9-migration].
+
+## Install Pantheon Search With Solr 8 on Drupal 9
+
+### Install the Search API Pantheon Module
+
+1. Navigate to the project's root directory on your local computer. If you have access to [Multidev](/multidev), checkout a new branch from master:
+
+  ```bash{promptUser: user}
+  git checkout -b solr master
+  ```
+
+  Otherwise, continue from the master branch.
+
+1. Add the Search API Pantheon module as a required dependency:
+
+  ```bash{promptUser: user}
+  composer require "drupal/search_api_pantheon ~1.0" --prefer-dist
+  ```
+
+1. You should now have the Search API Pantheon module installed along with its dependencies. Run `git status` to make sure you see the expected result. Commit and push the changes:
+
+  <TabList>
+
+  <Tab title="Without Multidev" id="install-nomulti" active={true}>
+
+  ```bash{promptUser: user}
+  git commit -am "Require drupal/search_api_pantheon ~1.0"
+  git push origin master
+  ```
+
+  </Tab>
+
+  <Tab title="With Multidev" id="install-multidev">
+
+  ```bash{promptUser: user}
+  git commit -am "Require drupal/search_api_pantheon ~1.0"
+  git push origin solr
+  terminus multidev:create <site>.dev solr
+  ```
+
+  </Tab>
+
+  </TabList>
+
+## Configure Solr
+
+To configure the connection with Pantheon, set the [connection mode](/sftp/#sftp-mode) to SFTP and complete the following steps.
+
+### Enable Modules
+
+Enable the Search API Pantheon module via the [Drupal interface](https://www.drupal.org/docs/8/extending-drupal-8/installing-contributed-modules-find-import-enable-configure-drupal-8#enable_your_mod). When prompted, click **Continue** to enable the [Search API](https://www.drupal.org/project/search_api) and [Search API Solr](https://www.drupal.org/project/search_api_solr) modules:
+
+### Disable Drupal Core's Search Module (Optional)
+
+If you are using Search API, then you probably will not be using Drupal Core's Search module. Uninstall the Search core module from `/admin/modules/uninstall` to avoid confusion in further configuration steps.
+
+### Add The Search Server
+
+Navigate to  `/admin/config/search/search-api/add-server` and configure the following, then click **Save**:
+
+- Server name: Pantheon
+
+- Backend: Solr
+
+- Solr Connector: Pantheon
+
+- Schema file: `modules/search_api_solr/solr-conf/4.x/schema.xml` (recommended)
+
+You can name the server anything you want but using something like "Pantheon" is a good way to remember where the connection goes. The Search API module provides schema files for each version of Solr (4, 5, and 6). You can customize schema files by copying these examples to your own custom module and editing them. If you are just getting started, we recommend selecting the file for Solr 4.
```

Comment:
I don't think we want them solecting the schema file for Solr 4 anymore. But we should test the updated schema to be sure.